pkg/cli/interactive_tests/test_timing.tcl

```diff
@@ -12,32 +12,49 @@ send "SELECT pg_sleep(0.02) FROM vehicles LIMIT 1;\r"
 eexpect "1 row"
 eexpect "execution"
 eexpect "network"
+eexpect root@
 
 # Ditto with multiple statements on one line
 send "SELECT * FROM vehicles LIMIT 1; CREATE TABLE t(a int);\r"
 eexpect "1 row"
 eexpect "Note: timings for multiple statements on a single line are not supported"
+eexpect root@
 end_test
 
 start_test "Test show_server_execution_times works correctly"
 send "\\set show_server_times=false\r"
 send "SELECT pg_sleep(0.02) FROM vehicles LIMIT 1;\r"
 eexpect "\nTime:"
+eexpect root@
 send "\\set show_server_times=true\r"
 send "SELECT pg_sleep(0.02) FROM vehicles LIMIT 1;\r"
 eexpect "execution"
 eexpect "network"
+eexpect root@
+end_test
+
+start_test "Check that server times also work if IntervalStyle is different"
+# regression test for issue #67618.
+send "set IntervalStyle = 'iso_8601';\r"
```

Comment:
mind throwing in a test for `sql_standard` as well?

Reply:
Done